sda-orch/templates/deployment.yaml

```diff
@@ -0,0 +1,147 @@
+        command: ["/bin/sh", "-c"]
+        args: ["/bin/cp /tls-certs/* /tls/ && chown 1000:1000 /tls/* && chmod 0400 /tls/*"]
+        securityContext:
+          allowPrivilegeEscalation: false
+        volumeMounts:
+        - name: tls-certs
+          mountPath: /tls-certs
+        - name: tls
+          mountPath: /tls
+    {{- end }}
+      containers:
+      - name: orchestrator
+        image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+        imagePullPolicy: {{ .Values.image.imagePullPolicy | quote }}
+        securityContext:
+          allowPrivilegeEscalation: false
+        env:
+        - name: LOG_LEVEL
+          value: {{ .Values.LogLevel | quote }}
+        - name: BROKER_PASSWORD
+          valueFrom:
+              secretKeyRef:
+                name: {{ template "sda.fullname" . }}-orchestrate
+                key: mqPassword
+        - name: BROKER_USER
+          value: {{ .Values.broker.username | quote }}
+        - name: DB_IN_PASSWORD
+          valueFrom:
+              secretKeyRef:
+                name: {{ template "sda.fullname" . }}-orchestrate
+                key: dbInPass
+        - name: DB_OUT_PASSWORD
+          valueFrom:
+              secretKeyRef:
+                name: {{ template "sda.fullname" . }}-orchestrate
+                key: dbOutPass
+        - name: DB_IN_USER
+          valueFrom:
+              secretKeyRef:
+                name: {{ template "sda.fullname" . }}-orchestrate
+                key: dbInUser
+        - name: DB_OUT_USER
+          valueFrom:
+              secretKeyRef:
+                name: {{ template "sda.fullname" . }}-orchestrate
+                key: dbOutUser
+        - name: DB_HOST
+          value: {{ required "A valid DB host is required" .Values.db.host | quote }}
+        - name: DB_DATABASE
+          value: {{ default "lega" .Values.db.name | quote }}
+        - name: DB_PORT
+          value: {{ .Values.db.port | quote }}
+        - name: DB_SSLMODE
+          value: {{ .Values.db.sslMode | quote }}
+        - name: BROKER_HOST
+          value: {{ required "A valid MQ Host is required" .Values.broker.host | quote }}
+        - name: BROKER_PORT
+          value: {{ .Values.broker.port | quote }}
+        - name: BROKER_VHOST
+          value: {{ .Values.broker.vhost | quote }}
+        - name: BROKER_EXCHANGE
+          value: {{ .Values.broker.exchange | quote }}
+        - name: VERIFIED_QUEUE
+          value: {{ .Values.broker.queue.verify | quote }}
+        - name: COMPLETED_QUEUE
+          value: {{ .Values.broker.queue.complete | quote }}
+        - name: INBOX_QUEUE
+          value: {{ .Values.broker.queue.inbox | quote }}
+        - name: FILES_QUEUE
+          value: {{ .Values.broker.queue.files | quote }}
+        - name: STABLEIDS_QUEUE
+          value: {{ .Values.broker.queue.stableid | quote }}
+        {{- if .Values.sslmode.ssl }}
+        - name: SSL_CACERT
+          value: {{ include "tlsPath" . }}/ca.crt
```

Comment:
I still haven't fully grasped this, are we certain quote isn't needed for these?

Reply:
reused what was in other charts and in my tests seems to work

Reply:
seems like we have multiple variants of that:

- https://github.com/neicnordic/sda-helm/blob/master/sda-svc/templates/auth-deploy.yaml#L117-L120
- https://github.com/neicnordic/sda-helm/blob/master/sda-svc/templates/ingest-deploy.yaml#L129-L132

I suggest we address this in a different PR.